content/docs/sidebar.json

```
@@ -525,5 +525,33 @@
         "slug": "dvclive-with-dvc"
       }
     ]
+  },
+  {
+    "label": "Studio",
+    "slug": "studio",
+    "source": "studio/index.md",
+    "icon": "studio",
+    "children": [
+      {
+        "label": "Get started",
+        "slug": "get-started"
+      },
+      {
+        "label": "Create a view",
+        "slug": "create-view"
+      },
+      {
+        "label": "Visualize experiments",
+        "slug": "visualize-exp"
+      },
+      {
+        "label": "Run experiements",
+        "slug": "experiments"
+      },
```

Let's use just

```suggestion
      "run-experiments",
```

But this requires changing the corresponding MD file's name too.